deploy/docker-swarm/clickhouse-setup/otel-collector-config.yaml

```diff
@@ -22,7 +22,7 @@ receivers:
       network:
 processors:
   batch:
-    send_batch_size: 1000
+    send_batch_size: 10000
```

Comment:
```
processors:
  batch:
    timeout: 10s
    send_batch_size: 10000
    send_batch_max_size: 11000
```

`send_batch_max_size` is the upper limit of the batch size which ensures that larger batches are split into smaller units.

Ref:
https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/batchprocessor#readme

Reply:
Will `send_batch_max_size` be used if we already specify `send_batch_size`?

Reply:
yes, it is.
`send_batch_size` would be the lower limit, while `send_batch_max_size` would be the upper limit.

Reply:
added